deep-learning/_posts/2018-08-22-deep-watershed.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
@czotti the article did not show up in the recent posts, because it will be published in the future :)
On August 22, 2018, to be precise!

Fred, rename this file please! haha